src/utils/components/AlertsCard/utils/utils.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { TFunction } from 'i18next';
+
+import {
+  ALL_ALERTS,
+  VIRTUALIZATION_ONLY_ALERTS,
+} from '@kubevirt-utils/components/AlertsCard/utils/constants';
+import { AlertType, SimplifiedAlerts } from '@kubevirt-utils/components/AlertsCard/utils/types';
+import {
+  PrometheusLabels,
+  YellowExclamationTriangleIcon,
+} from '@openshift-console/dynamic-plugin-sdk';
+import {
+  BlueInfoCircleIcon,
+  RedExclamationCircleIcon,
+} from '@openshift-console/dynamic-plugin-sdk/lib/app/components/status/icons';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-triangle-icon';
+
+export const color = {
+  [AlertType.critical]: 'red',
+  [AlertType.warning]: 'orange',
+  [AlertType.info]: 'blue',
+};
+
+export const icon = {
+  [AlertType.critical]: <ExclamationCircleIcon />,
+  [AlertType.warning]: <ExclamationTriangleIcon />,
+  [AlertType.info]: <ExclamationCircleIcon />,
+};
+
+export const filterAlerts = (sortedAlerts: SimplifiedAlerts) => {
+  const filteredAlerts = { critical: [], warning: [], info: [] };
+
+  for (const [key, value] of Object.entries(sortedAlerts)) {
+    filteredAlerts[key] = value.filter((alert) => !alert?.isVMAlert);
+  }
+
+  return filteredAlerts;
+};
+
+export const getAlertIcon = (alertType: AlertType) => {
```

Comment:
can be mapped into an object saving function call

Reply:
Fixed!